R/class_vaccination.R

```diff
@@ -132,9 +170,9 @@ validate_daedalus_vaccination <- function(x) {
   }
 
   # check class members
-  expected_invariants <- c("name", daedalus.data::vaccination_parameter_names)
+  expected_invariants <- c(daedalus.data::vaccination_parameter_names)
```

Comment:
```suggestion
  expected_invariants <- daedalus.data::vaccination_parameter_names
```

surprised that lintr doesn't pick that up.  Also, as above, probably `attributes` and not `invariants`